server/utils/sortHeader.ts

```diff
@@ -27,7 +27,11 @@ export const sortHeader = <T extends string>(
   }
 
   return {
-    html: `<a href="${basePath}?${createQueryString({ ...qsArgs, sortBy: targetField, sortDirection })}">${text}</a>`,
+    html: `<a href="${basePath}?${createQueryString({
+      ...qsArgs,
+      sortBy: targetField,
+      sortDirection,
+    })}" title="Sort by ${text}">${text}</a>`,
```

Comment:
Could add ariaSort to the title to indicate the link's direction of sort.

Reply:
Superseded by different approach to announcing sortable headers!